cmd/genkubedocs/gen_kube_docs.go

```diff
@@ -34,6 +36,7 @@ import (
 )
 
 func main() {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Why are we using a variable seed to generate docs? I don't know how it is used, but I would expect that we want deterministic output for gen_kube_docs?

Reply:
Random numbers probably shouldn't be used to generate docs? Or if they are, then why not really random? I think properly seeding RNG is a good way to catch any issues.

Reply:
Agree that they probably shouldn't be used, but if they are you want consistent results. Otherwise, every time I need to rebuild the docs, I'll end up with a bunch of unrelated changes (breaking caching, among other things).

Reply:
I think the right way would be to use an RNG seeded to a fixed number explicitly, rather than relying on the global one being not initialized. The only way to find out if this is actually an issue is to keep it here, but I'll remove it if you want it that way, np.

Reply:
@ash2k agreed, fixed seed SGTM

Reply:
Sorry, I was not clear enough. I meant that the code, that uses RNG and wants a fixed number sequence (if such code exists in this docs generator at all), should be using a non-global RNG and should seed it with some constant. It should not be relying on the global RNG to be seeded to a fixed number (because some other part of the program may have the opposite need).

In other words:
> Agree that they probably shouldn't be used, but if they are you want consistent results. Otherwise, every time I need to rebuild the docs, I'll end up with a bunch of unrelated changes (breaking caching, among other things).

My answer is that it'd be bug and it should be found and fixed instead of seeding global RNG with a fixed number to work it around.

I'll just remove this line from here because it does not matter in practice most likely and we cannot seem to agree on this one :)

Reply:
FWIW, doc generation which uses random numbers need to be found and fixed, so I support making it actually random.